test/cppsuite/tests/reverse_split.cpp

```diff
@@ -128,6 +65,8 @@ class reverse_split : public test {
         scoped_cursor write_cursor = std::move(tc->session.open_scoped_cursor(coll.name));
 
         while (tc->running()) {
+            /* Synchronise across our truncate threads so they don't diverge over time. */
```

Comment:
What do you mean by diverge?

Reply:
Lets say there are 5 truncate threads running, they will all sleep the specified amount every iteration however their actions can take different amounts of time. So when the test begins they all call truncate at the same time and by the time the test ends they are calling truncate at varied times. 

The synchronize code ensure they all call truncate at roughly the same time for the duration of the test which results in a spikier workload.